Optimised getCell() and added getRange()

Jonowa opened the pull request:
I think my version of ->getCell() is even much faster than using xpath in case of big worksheets. (My test scenery: worksheet with 25 columns and about 5000 rows with 1000 cycles => my version took about 0.008 sec, the use of xpath took 1.5 sec)
Moreover I've added a function ->getRange()
The function _num2name() is obsolete.

shuchkin commented:
try your getCell with empty row

123|456|789
      |      |      
321|654|987

Jonowa commented:
Hi Sergey,
you are right. I've not testet it on empty rows or invalid indexes (like "?1" or "AA"). I've changed the function `->getCell()` back to your version. I've also changed my added function `->getRange()` using `->xpath()`. You can find my changes in my fork.
Regards
Joerg

shuchkin commented:
not ready to production
